package/MDAnalysis/core/universe.py

```diff
@@ -158,7 +157,7 @@ class Universe(object):
     in_memory
         After reading in the trajectory, transfer it to an in-memory
         representations, which allow for manipulation of coordinates.
-    in_memory_step
+    in_memory_frame_interval
```

Comment:
Your working on an outdated version. Please rebase against the latest version of develop.